src/main/java/jenkins/plugins/office365connector/WebhookJobPropertyDescriptor.java

```diff
@@ -73,7 +73,16 @@ public WebhookJobProperty newInstance(StaplerRequest req, JSONObject formData) t
                     webhooks.addAll(req.bindJSONToList(Webhook.class, webhooksArrayData));
                 } else {
                     JSONObject webhooksObjectData = (JSONObject) webhooksData;
-                    webhooks.add(req.bindJSON(Webhook.class, webhooksObjectData));
+                    Boolean isCompactNotificationOn = webhooksObjectData.getBoolean("compactNotification");
```

Comment:
I'm not sure what is going on here

Reply:
this should not be needed, with the proper data-bound setter.

Reply:
Removed. Simply forgot about this one